packages/dd-trace/test/node_modules/express-mock/package.json

```diff
@@ -2,7 +2,6 @@
   "name": "express-mock",
   "version": "4.0.0",
   "description": "",
-  "main": "index.js",
```

Comment:
Why remove this? The entrypoint is still `index.js` for now and I would prefer to stay explicit about it.

Reply:
I thought in two things when deleting that line: to test `pkg.main || 'index.js'` on instrumenter.js, and because this seems more representative to the real _package.json_ shipped by Express.

I understand your concern anyway and just push another way to tests this using the `other` package available on the tests. I used it because I couldn't find any use on the tests for that mock and I don't want to add another plugin mock just for this case.

Do you agree with this approach? If not please point me in the right direction.
Thanks!

Reply:
Definitely agree with this approach! The more we can test with less mocks the better.